lib/loggery/sidekiq/setup.rb

```diff
@@ -4,9 +4,9 @@ module Loggery
   module Sidekiq
     module Setup
       def self.setup
-        ::Sidekiq::Logging.logger = Rails.logger
-
         ::Sidekiq.configure_server do |config|
+          config.logger = Rails.logger
```

Comment:
Just out of curiosity: will this work with Sidekiq < 6 as well?

Reply:
it does